Source/AwsCommonRuntimeKit/mqtt/Mqtt5Client.swift

```diff
@@ -286,7 +286,7 @@ public class Mqtt5ClientCore {
         try self.rwlock.read {
             // Validate close() has not been called on client.
             guard let rawValue = self.rawValue else {
-                throw CommonRunTimeError.crtError(CRTError.makeFromLastError())
+                throw CommonRunTimeError.crtError(CRTError(code: AWS_CRT_SWIFT_MQTT_CLIENT_CLOSED.rawValue))
```

Comment:
Debatable: Probably better to just use `AWS_ERROR_INVALID_STATE` with context?

Reply:
I decided to keep the Swift error space, as it makes sense to maintain it whether we use it or not.  Consequently, I am also keeping the `AWS_CRT_SWIFT_MQTT_CLIENT_CLOSED` error code. 
Additionally, I removed `AWS_CRT_SWIFT_INVALID_ARGUMENT ` to avoid confusion with `AWS_ERROR_INVALID_ARGUMENT `